update snowflake import documentation with key pair auth troubleshooting (#605)
amplitude/amplitude-docs: content/collections/source-catalog/en/snowflake.md

- **Cause**: This occurs when the Snowflake role used for the integration no longer has permission to access the `INFORMATION_SCHEMA` of the specified Snowflake database. Amplitude requires access to this schema to check the status of queries running on the customer's Snowflake instance.
- **Solution**: Ensure that the role used for the integration has the necessary permissions to access the `INFORMATION_SCHEMA` in the target Snowflake database. This issue often arises if the role was recently changed or updated without the appropriate access being maintained.

> 2. **Error: `JWT token is invalid.`**

Reviewer: ```suggestion
2. Error: `JWT token is invalid.`
```

- **Cause**: This error appears when there is a mismatch between the public key attached to the given user and the private key Amplitude generates for key pair authentication.

> - **Solution**: Ensure that the public key is properly set on the given user, and that the account is provided in the format `ORGNAME-ACCOUNTNAME`. If the account is provided with an account locator at the end, key pair authentication is not guaranteed to succeed, even if the public key is set properly.

Reviewer: ```suggestion
- **Solution**: Ensure that the public key is properly set on the given user, and that the account is provided in the format `ORGNAME-ACCOUNTNAME`. If the account has an account locator at the end, key pair authentication may not succeed, even if the public key is set properly.
```

Can we provide an example of what an account locator appended to the end looks like?

Author: Instead of providing an example of an [account locator](https://docs.snowflake.com/en/user-guide/admin-account-identifier#finding-the-region-and-locator-for-an-account), I elected to provide some SQL that the customer should run that is guaranteed to produce the correct format.
